Wordle.Api/Wordle.Api/Controllers/LeaderboardController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class LeaderboardController : ControllerBase
+    {
+        private readonly LeaderboardService _leaderboardService;
+
+        public LeaderboardController(LeaderboardService leaderboardService)
+        {
+            _leaderboardService = leaderboardService;
+        }
+
+        [HttpPost("AddNewScore")]
+        public async Task<Player> AddNewScore(string name, int attempts)
+        {
+            //Add the players score to the database
+            return await _leaderboardService.AddNewScore(name, attempts);
+
+        }
+
+        [HttpGet("GetTopTenScores")]
```

Comment:
Typically we don't put the action in the name
```suggestion
        [HttpGet("TopTenScores")]
```